ecl/hqlcpp/hqlregex.cpp

```diff
@@ -1524,6 +1544,8 @@ void HqlRegexExpr::generateRegex(GenerateRegexCtx & ctx)
                         created.setown(new RegexAsciiIPattern(len, data));
                     break;
                 }
+            default:
+                UNIMPLEMENTED;
```

Comment:
@richardkchapman I moved the appropriate errors into the case branches - this is UNIMPLEMENTED is correct